module/Activity/view/email/activity_created_require_GEFLITST.phtml

```diff
@@ -0,0 +1,36 @@
+<?php $lang = 'en'; ?>
+For English, see below.
+
+L.s.<br /><br />
+
+Er is een nieuwe activiteit aangemaakt op de GEWIS website waarvoor een fotograaf van GEFLITST nodig is.
+
+<h3><?= $this->escapeHtml($activity->getName()) ?></h3>
+
+<p><?= $this->escapeHtml($activity->getDescription()) ?></p>
+
+<br/>
+
+Het orgaan dat deze activiteit heeft aangemaakt kan worden bereikt via dit e-mailadres: <?= $this->escapeHtml($activity->getResourceOrgan()->getOrganInformation()['email']) ?>
```

Comment:
Not all activities have an organ associated with them, it might be better to use the email of the member whom created the activity.